calico-enterprise_versioned_docs/version-3.16/_includes/release-notes/_v3.16.2-release-notes.mdx

```diff
@@ -0,0 +1,5 @@
+xx June 2023
+
+:::note
+Release 3.16.2 was incorrectly tagged during the release process so 3.16.3 is the next patch release after 3.16.1. (There is no 3.16.2 patch release).
```

Comment:
I wonder whether this explanation is unnecessarily detailed. The parenthetical comment seems enough to me (as does nothing!).